doc/howto/ubuntu-image-classic-examples.rst

```diff
@@ -0,0 +1,269 @@
+.. _how-to-use-ubuntu-image-with-the-classic-command:
+
+How to use ``ubuntu-image`` with the ``classic`` command
+========================================================
+
+The :command:`classic` command in :command:`ubuntu-image` is used to create classical Ubuntu images based on traditional Ubuntu releases (like Desktop or Server images). The :command:`classic` command uses an **image definition** YAML file to define the image structure.
+
+.. note:: :command:`ubuntu-image` requires elevated permissions to run correctly. Use :command:`sudo` when running the command.
+
+
+.. _general-command-structure:
+
+General command structure
```

Comment:
Will this section be repeated in the future "ubuntu image with the snap command" how-to?

Reply:
It pbbly should. In which case I'd create a snippet to single-source it, changing the subcommand only.

Speaking of which: I only worked on the `classic` examples for now because the use of `snap`  subcommand is described in https://ubuntu.com/core/docs/build-an-image -- pending a decision on how best to integrate these docs, I refrained from duplicating that example.

Which reminds me -- I wanted to add a link to that article here.